Update websockets.mdx (#18280)
cloudflare/cloudflare-docs: src/content/docs/network/websockets.mdx

Investigating issues with Websocket can be facilitated with client tools like [wscat](https://github.com/websockets/wscat).
Being able to reproduce an issue on a single URL with a minimalistic tool helps narrowing down the issue.

> When looking at [HTTP requests logs](/logs/reference/log-fields/zone/http_requests/) note that the EdgeStartTimestamp and EdgeStopTimestamp fields represent the duration of the WebSocket connection and is not just a representation of the initial HTTP connection.

Reviewer: ```suggestion
The `EdgeStartTimestamp` and `EdgeStopTimestamp` fields in [HTTP requests logs](/logs/reference/log-fields/zone/http_requests/) represent the duration of the WebSocket connection (they do not represent the initial HTTP connection).
```
